packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -117,7 +123,7 @@ const AccountTransferForm = observer(
 
         const [from_accounts, setFromAccounts] = React.useState({});
         const [to_accounts, setToAccounts] = React.useState({});
-        const [transfer_to_hint, setTransferToHint] = React.useState<string>();
+        const [transfer_to_hint, setTransferToHint] = React.useState<JSX.Element>();
```

Comment:
do we need to set type for this useeffect?

Reply:
yes, sine we are not passing a default value. specifying the type to be `<JSX.Element>` would be better than passing a `<Localize i18n_default_text="">` to useState initially, right?